integration_tests/pom.xml

```diff
@@ -211,32 +211,6 @@
                     <classifier>${spark.version.classifier}</classifier>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
@pxLi and @tgravescs git blame indicates that the two of you touched this. It looks like it was added in on purpose so we get version info in both jars. Is that correct?

Reply:
Yes this is to make sure all jars have version properties which we then use later to make sure all the jars we pull are built from same version of source code, please put it back.

Reply:
Reverted.